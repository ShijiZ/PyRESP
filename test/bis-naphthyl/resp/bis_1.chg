All values are reported in atomic units
%FLAG TITLE
 molecule 1: naphthyl

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
   1  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
   2  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
   3  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
   4  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
   5  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
   6  -7.2380090E+00   1.1409360E+00   1.9545440E-01
   7  -4.5750250E+00   1.4616850E+00   1.5977450E-01
   8  -3.3997450E+00   3.8559500E+00   4.4761770E-01
   9  -8.4462660E-01   4.0664900E+00   3.9297250E-01
  10   7.6098920E-01   1.9305880E+00   4.8666140E-02
  11   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
  12  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
  13  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
  14  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
  15  -8.4208230E+00   2.7785030E+00   4.4958490E-01
  16  -4.5649300E+00   5.5057230E+00   7.0573740E-01
  17   1.3078800E-02   5.9002260E+00   6.1156470E-01
  18   3.5994910E+00   2.3574250E+00   2.7072230E-02
  19   5.2326990E+00   2.5578210E-01  -1.1285470E+00
  20   7.8846260E+00   1.2812330E+00  -1.6756280E+00
  21   5.4574950E+00  -2.0694230E+00   6.7427150E-01
  22   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
  23   6.3981970E+00  -1.6028000E+00   2.7540580E+00
  24   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
  25   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
  26   8.7740430E+00   1.9007740E+00   6.2340200E-02
  27   7.8358690E+00   2.8633100E+00  -2.9949260E+00
  28   4.2425500E+00   2.6484730E+00   1.9549130E+00
  29   3.9400330E+00   4.1246770E+00  -9.7823600E-01

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
   1          6          0    -1.5599986E-01
   2          6          0     9.8561122E-02
   3          6          0    -1.8770557E-01
   4          6          0    -1.8005028E-01
   5          6          0    -1.4938600E-01
   6          6          0    -2.2901668E-01
   7          6          0     1.6781871E-01
   8          6          0    -3.0892870E-01
   9          6          0    -1.3014956E-01
  10          6          0    -1.6849368E-02
  11          1          0     1.2656996E-01
  12          1          0     1.5768343E-01
  13          1          0     1.2813541E-01
  14          1          0     1.2182813E-01
  15          1          0     1.2790716E-01
  16          1          0     1.4891236E-01
  17          1          0     1.1272398E-01
  18          6          0     2.7238526E-02
  19          6          0     1.3328629E-01
  20          6          0    -1.9464205E-01
  21          6          0     7.7809709E-01
  22          8          0    -7.9516175E-01
  23          8         22    -7.9516175E-01
  24          1          0    -7.3737608E-02
  25          1          0     4.2493795E-02
  26          1          0     3.0495529E-02
  27          1          0    -2.0084057E-03
  28          1          0     3.1979868E-02
  29          1          0    -1.4933769E-02

%FLAG TITLE
 molecule 2: naphthyl

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
  30  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
  31  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
  32  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
  33  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
  34  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
  35  -7.2380090E+00   1.1409360E+00   1.9545440E-01
  36  -4.5750250E+00   1.4616850E+00   1.5977450E-01
  37  -3.3997450E+00   3.8559500E+00   4.4761770E-01
  38  -8.4462660E-01   4.0664900E+00   3.9297250E-01
  39   7.6098920E-01   1.9305880E+00   4.8666140E-02
  40   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
  41  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
  42  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
  43  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
  44  -8.4208230E+00   2.7785030E+00   4.4958490E-01
  45  -4.5649300E+00   5.5057230E+00   7.0573740E-01
  46   1.3078800E-02   5.9002260E+00   6.1156470E-01
  47   3.5994910E+00   2.3574250E+00   2.7072230E-02
  48   5.2326990E+00   2.5578210E-01  -1.1285470E+00
  49   7.8846260E+00   1.2812330E+00  -1.6756280E+00
  50   5.4574950E+00  -2.0694230E+00   6.7427150E-01
  51   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
  52   6.3981970E+00  -1.6028000E+00   2.7540580E+00
  53   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
  54   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
  55   8.7740430E+00   1.9007740E+00   6.2340200E-02
  56   7.8358690E+00   2.8633100E+00  -2.9949260E+00
  57   4.2425500E+00   2.6484730E+00   1.9549130E+00
  58   3.9400330E+00   4.1246770E+00  -9.7823600E-01

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
  30          6          1    -1.5599986E-01
  31          6          2     9.8561122E-02
  32          6          3    -1.8770557E-01
  33          6          4    -1.8005028E-01
  34          6          5    -1.4938600E-01
  35          6          6    -2.2901668E-01
  36          6          7     1.6781871E-01
  37          6          8    -3.0892870E-01
  38          6          9    -1.3014956E-01
  39          6         10    -1.6849368E-02
  40          1         11     1.2656996E-01
  41          1         12     1.5768343E-01
  42          1         13     1.2813541E-01
  43          1         14     1.2182813E-01
  44          1         15     1.2790716E-01
  45          1         16     1.4891236E-01
  46          1         17     1.1272398E-01
  47          6         18     2.7238526E-02
  48          6         19     1.3328629E-01
  49          6         20    -1.9464205E-01
  50          6         21     7.7809709E-01
  51          8         22    -7.9516175E-01
  52          8         22    -7.9516175E-01
  53          1         24    -7.3737608E-02
  54          1         25     4.2493795E-02
  55          1         26     3.0495529E-02
  56          1         27    -2.0084057E-03
  57          1         28     3.1979868E-02
  58          1         29    -1.4933769E-02

