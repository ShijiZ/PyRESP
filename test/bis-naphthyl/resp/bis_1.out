
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
 test for bis-naphthyl
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 2   iqopt       = 1
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00050000

 --------------------
 Molecule Information
 --------------------
 Multiple-molecule run of 2 molecules

 Molecule 1 weight:     1.000
 Molecule 1 Name: naphthyl
 Total charge (ich): -1
 Number of centers: 29
    1    6    0
    2    6    0
    3    6    0
    4    6    0
    5    6    0
    6    6    0
    7    6    0
    8    6    0
    9    6    0
   10    6    0
   11    1    0
   12    1    0
   13    1    0
   14    1    0
   15    1    0
   16    1    0
   17    1    0
   18    6    0
   19    6    0
   20    6    0
   21    6    0
   22    8    0
   23    8   22
   24    1    0
   25    1    0
   26    1    0
   27    1    0
   28    1    0
   29    1    0

 Molecule 2 weight:     1.000
 Molecule 2 Name: naphthyl
 Total charge (ich): -1
 Number of centers: 29
   30    6    0
   31    6    0
   32    6    0
   33    6    0
   34    6    0
   35    6    0
   36    6    0
   37    6    0
   38    6    0
   39    6    0
   40    1    0
   41    1    0
   42    1    0
   43    1    0
   44    1    0
   45    1    0
   46    1    0
   47    6    0
   48    6    0
   49    6    0
   50    6    0
   51    8    0
   52    8   22
   53    1    0
   54    1    0
   55    1    0
   56    1    0
   57    1    0
   58    1    0
 ---------------------------------------------------------
 Inter-Molecular Charge Constraint(s) Info
 ---------------------------------------------------------
    1   19    1   20    1   21    1   22    1   23    1   24    1   25    1   26
    1   27    2   20    2   25    2   26    2   27

 -------------------------------
 Charges Equivalence Information 
 -------------------------------
    1    1    2    1
    1    2    2    2
    1    3    2    3
    1    4    2    4
    1    5    2    5
    1    6    2    6
    1    7    2    7
    1    8    2    8
    1    9    2    9
    1   10    2   10
    1   11    2   11
    1   12    2   12
    1   13    2   13
    1   14    2   14
    1   15    2   15
    1   16    2   16
    1   17    2   17
    1   18    2   18
    1   19    2   19
    1   20    2   20
    1   21    2   21
    1   22    2   22
    1   23    2   23
    1   24    2   24
    1   25    2   25
    1   26    2   26
    1   27    2   27
    1   28    2   28
    1   29    2   29

 -----------------------------
 Atom Ivary Information
 -----------------------------
    6    0
    6    0
    6    0
    6    0
    6    0
    6    0
    6    0
    6    0
    6    0
    6    0
    1    0
    1    0
    1    0
    1    0
    1    0
    1    0
    1    0
    6    0
    6    0
    6    0
    6    0
    8    0
    8   22
    1    0
    1    0
    1    0
    1    0
    1    0
    1    0

    6    1
    6    2
    6    3
    6    4
    6    5
    6    6
    6    7
    6    8
    6    9
    6   10
    1   11
    1   12
    1   13
    1   14
    1   15
    1   16
    1   17
    6   18
    6   19
    6   20
    6   21
    8   22
    8   22
    1   24
    1   25
    1   26
    1   27
    1   28
    1   29

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =   58
 Weight factor on initial charge restraints=  0.000500
 There are  3 charge constraints

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =    29
 total number of esp points =  1250

 center       X               Y               Z
    1  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
    2  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
    3  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
    4  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
    5  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
    6  -7.2380090E+00   1.1409360E+00   1.9545440E-01
    7  -4.5750250E+00   1.4616850E+00   1.5977450E-01
    8  -3.3997450E+00   3.8559500E+00   4.4761770E-01
    9  -8.4462660E-01   4.0664900E+00   3.9297250E-01
   10   7.6098920E-01   1.9305880E+00   4.8666140E-02
   11   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
   12  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
   13  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
   14  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
   15  -8.4208230E+00   2.7785030E+00   4.4958490E-01
   16  -4.5649300E+00   5.5057230E+00   7.0573740E-01
   17   1.3078800E-02   5.9002260E+00   6.1156470E-01
   18   3.5994910E+00   2.3574250E+00   2.7072230E-02
   19   5.2326990E+00   2.5578210E-01  -1.1285470E+00
   20   7.8846260E+00   1.2812330E+00  -1.6756280E+00
   21   5.4574950E+00  -2.0694230E+00   6.7427150E-01
   22   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
   23   6.3981970E+00  -1.6028000E+00   2.7540580E+00
   24   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
   25   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
   26   8.7740430E+00   1.9007740E+00   6.2340200E-02
   27   7.8358690E+00   2.8633100E+00  -2.9949260E+00
   28   4.2425500E+00   2.6484730E+00   1.9549130E+00
   29   3.9400330E+00   4.1246770E+00  -9.7823600E-01


 Reading esp's for molecule   2
 total number of atoms      =    29
 total number of esp points =  1250

 center       X               Y               Z
    1  -3.4196870E-01  -3.8245240E-01  -2.2379470E-01
    2  -3.0137120E+00  -6.6733250E-01  -1.7237140E-01
    3  -4.1655650E+00  -3.0774580E+00  -4.5241950E-01
    4  -6.7218550E+00  -3.3337600E+00  -4.0980620E-01
    5  -8.2871570E+00  -1.1876780E+00  -8.2104850E-02
    6  -7.2380090E+00   1.1409360E+00   1.9545440E-01
    7  -4.5750250E+00   1.4616850E+00   1.5977450E-01
    8  -3.3997450E+00   3.8559500E+00   4.4761770E-01
    9  -8.4462660E-01   4.0664900E+00   3.9297250E-01
   10   7.6098920E-01   1.9305880E+00   4.8666140E-02
   11   8.0949660E-01  -2.0403480E+00  -4.5754250E-01
   12  -2.9636740E+00  -4.6991780E+00  -6.9257350E-01
   13  -7.5698620E+00  -5.1703360E+00  -6.2116640E-01
   14  -1.0308890E+01  -1.4128860E+00  -5.1018840E-02
   15  -8.4208230E+00   2.7785030E+00   4.4958490E-01
   16  -4.5649300E+00   5.5057230E+00   7.0573740E-01
   17   1.3078800E-02   5.9002260E+00   6.1156470E-01
   18   3.5994910E+00   2.3574250E+00   2.7072230E-02
   19   5.2326990E+00   2.5578210E-01  -1.1285470E+00
   20   7.8846260E+00   1.2812330E+00  -1.6756280E+00
   21   5.4574950E+00  -2.0694230E+00   6.7427150E-01
   22   4.6853510E+00  -4.1020210E+00  -1.7990010E-01
   23   6.3981970E+00  -1.6028000E+00   2.7540580E+00
   24   4.3903760E+00  -3.6916190E-01  -2.8929710E+00
   25   9.0685060E+00  -1.8308430E-01  -2.4959300E+00
   26   8.7740430E+00   1.9007740E+00   6.2340200E-02
   27   7.8358690E+00   2.8633100E+00  -2.9949260E+00
   28   4.2425500E+00   2.6484730E+00   1.9549130E+00
   29   3.9400330E+00   4.1246770E+00  -9.7823600E-01


 ------------------------
 Optimization Information
 ------------------------
 iqopt=1, all q0 values will be set to 0

 Number of unique UNfrozen charge centers =    28

 Non-linear optimization requested.
  qchnge =    3.9669928382E-02
  qchnge =    4.3129973714E-03
  qchnge =    9.0800131022E-04
  qchnge =    2.5611918505E-04
  qchnge =    8.4491946199E-05
  qchnge =    3.4003746120E-05
  qchnge =    1.6298793306E-05
  qchnge =    8.5472161825E-06
  qchnge =    4.6329021893E-06
  qchnge =    2.5384471231E-06
  qchnge =    1.3957482392E-06
  qchnge =    7.6836517630E-07

 Convergence in   11 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1: naphthyl
 molecule 2: naphthyl

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         6        0.000000     -0.156000      0       0.002698
    2         6        0.000000      0.098561      0       0.003561
    3         6        0.000000     -0.187706      0       0.002351
    4         6        0.000000     -0.180050      0       0.002428
    5         6        0.000000     -0.149386      0       0.002781
    6         6        0.000000     -0.229017      0       0.002001
    7         6        0.000000      0.167819      0       0.002560
    8         6        0.000000     -0.308929      0       0.001540
    9         6        0.000000     -0.130150      0       0.003046
   10         6        0.000000     -0.016849      0       0.004931
   11         1        0.000000      0.126570      0       0.000000
   12         1        0.000000      0.157683      0       0.000000
   13         1        0.000000      0.128135      0       0.000000
   14         1        0.000000      0.121828      0       0.000000
   15         1        0.000000      0.127907      0       0.000000
   16         1        0.000000      0.148912      0       0.000000
   17         1        0.000000      0.112724      0       0.000000
   18         6        0.000000      0.027239      0       0.004824
   19         6        0.000000      0.133286      0       0.003001
   20         6        0.000000     -0.194642      0       0.002285
   21         6        0.000000      0.778097      0       0.000637
   22         8        0.000000     -0.795162      0       0.000624
   23         8        0.000000     -0.795162     22       0.000624
   24         1        0.000000     -0.073738      0       0.000000
   25         1        0.000000      0.042494      0       0.000000
   26         1        0.000000      0.030496      0       0.000000
   27         1        0.000000     -0.002008      0       0.000000
   28         1        0.000000      0.031980      0       0.000000
   29         1        0.000000     -0.014934      0       0.000000

   30         6        0.000000     -0.156000      1       0.002698
   31         6        0.000000      0.098561      2       0.003561
   32         6        0.000000     -0.187706      3       0.002351
   33         6        0.000000     -0.180050      4       0.002428
   34         6        0.000000     -0.149386      5       0.002781
   35         6        0.000000     -0.229017      6       0.002001
   36         6        0.000000      0.167819      7       0.002560
   37         6        0.000000     -0.308929      8       0.001540
   38         6        0.000000     -0.130150      9       0.003046
   39         6        0.000000     -0.016849     10       0.004931
   40         1        0.000000      0.126570     11       0.000000
   41         1        0.000000      0.157683     12       0.000000
   42         1        0.000000      0.128135     13       0.000000
   43         1        0.000000      0.121828     14       0.000000
   44         1        0.000000      0.127907     15       0.000000
   45         1        0.000000      0.148912     16       0.000000
   46         1        0.000000      0.112724     17       0.000000
   47         6        0.000000      0.027239     18       0.004824
   48         6        0.000000      0.133286     19       0.003001
   49         6        0.000000     -0.194642     20       0.002285
   50         6        0.000000      0.778097     21       0.000637
   51         8        0.000000     -0.795162     22       0.000624
   52         8        0.000000     -0.795162     22       0.000624
   53         1        0.000000     -0.073738     24       0.000000
   54         1        0.000000      0.042494     25       0.000000
   55         1        0.000000      0.030496     26       0.000000
   56         1        0.000000     -0.002008     27       0.000000
   57         1        0.000000      0.031980     28       0.000000
   58         1        0.000000     -0.014934     29       0.000000

 Sum over the calculated charges:     -2.000

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          3.2877896E+01
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   5.5817720E-03
 The root-mean-squared error (RMSE = sqrt(RSS/N))               1.4942252E-03
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         1.3029689E-02

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1       12.81549  -10.77220    6.24370   -3.03524
 #MOL         D          Dx         Dy         Dz
   2       12.81549  -10.77220    6.24370   -3.03524

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X  -20.87790
        Y   17.86087   10.70907
        Z  -11.49663    4.47119   10.16883
 #MOL          X          Y          Z
   2    X  -20.87790
        Y   17.86087   10.70907
        Z  -11.49663    4.47119   10.16883

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   18.83391
        Y    0.00000   13.65089
        Z    0.00000    0.00000  -32.48480
 #MOL          X          Y          Z
   2    X   18.83391
        Y    0.00000   13.65089
        Z    0.00000    0.00000  -32.48480